boards/nucleo-u575zi-q/doc.txt

```diff
@@ -5,14 +5,10 @@
 
 ## Overview
 
-The Nucleo-L552ZE-Q is a board from ST's Nucleo family supporting the ARM Cortex-M33
+The Nucleo-U575ZI-Q is a board from ST's Nucleo family supporting the ARM Cortex-M33
 STM32U575ZIT6Q ultra-low-power microcontroller with TrustZone, 768KiB of RAM and 2MiB
 of Flash.
 
-## Hardware
-
-![Nucleo144 L552ZE-Q](https://www.st.com/bin/ecommerce/api/image.PF271812.en.feature-description-include-personalized-no-cpn-medium.jpg)
```

Comment:
Is this not the right picture?

Reply:
Is is the right picture, but it was the wrong caption. The picture could probably stay and the caption be corrected to "Nucleo-144 U575ZI-Q".